tests/unit/test_response.py

```diff
@@ -158,6 +158,15 @@ def stream(self, amt=1024, decode_content=None):
                 break
             yield chunk
 
+    def test_streaming_line_abstruse_newline_standard(self):
```

Comment:
It looks like the rebase got messed up here.  This test is now part of the `FakeRawResponse` class so it won't be run.  I think this was meant to be part of `TestStreamWrapper`.

Reply:
Oops, sorry for the messy rebase. Just made a new patch